### DISCUSSION TITLE
Feedback component

### DISCUSSION THREAD

**Discussion by dcaryll** [RedHat-UX/red-hat-design-system#334] [Ideas]
We're using Qualtrics to support feedback boxes across our website properties. 

There are issues with the ability to get this data into the right place. There are also restrictions as to how and what we can style. It's a relatively out of the box solution for feedback forms and other user feedback methods Qualtrics offers.

Was thinking 2 things: 

1. could we design our own experience _and_ 
2. could we build a tool that could handle this and could output the data into, say, Tableau?

**Comment by bennypowers** [comment]
yes and yes
